package.yaml

```diff
@@ -43,6 +43,12 @@ dependencies:
   - aeson
   - Cabal
   - extensible-data
+  - arithmetic-circuits
+  - galois-field
+  - crypto-numbers
+  - zkp
+  - pairing
+  - MonadRandom
```

Comment:
We should probably Comment that these came from arithCircuit backend.

Will probably need to merge upstream so all the comments on where what deps come from apply.

Reply:
They won't be here for the long term anyway, but I agree that we should comment about it so we know if/when we need to remove them.